login-workflow/src/screens/ForgotPasswordScreen/types.ts

```diff
@@ -3,76 +3,37 @@ import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.ty
 import { SuccessScreenProps } from '../SuccessScreen';
 
 export type ForgotPasswordScreenProps = WorkflowCardProps & {
-    // used to display contact phone number
-    contactPhone?: string;
-
-    // used to display response time
-    responseTime?: string;
-
-    // TODO: These need cleaned up to use the WorkflowCardProps instead
-
-    // used to update the instruction
-    description?: (responseTime: string) => React.ReactNode;
-
-    // used to update the screen title
-    title?: string;
-
-    // used to display back button
-    showBackButton?: boolean;
-
-    // used to display back button label
-    backButtonLabel?: string;
-
-    // used to enable to back button
-    canGoBack?: boolean | (() => boolean);
-
-    // used to display next button
-    showNextButton?: boolean;
-
-    // used to display next button label
-    nextButtonLabel?: string;
-
-    // used to enable to next button
-    canGoNext?: boolean | (() => boolean);
-
-    /**
-     * The label for the email field
-     */
+    // label for the textfield
     emailLabel?: string;
 
-    /**
-     * The initial value for the email text field
-     */
+    // used to pre-populate the email input field
     initialEmailValue?: string;
 
-    /**
-     * A function used to test the input for valid formatting
-     * @param {string} email - the provided email address
-     * @returns boolean | string
-     */
+    // used to test the input for valid formatting
     emailValidator?: (email: string) => boolean | string;
 
-    /**
-     * Used for ForgotPasswordScreen SuccessScreen
-     */
+    // used for each slot in `ForgotPasswordScreenBase`
     slots?: {
         SuccessScreen?: (props: SuccessScreenProps) => JSX.Element;
     };
 
-    /**
-     * Applied to slot from SuccessScreen
-     */
+    // props applied to each slot
     slotProps?: {
```

Comment:
update the documentation format to be similar to other type files

/**
     * A function used to test the input for valid formatting
     * @param {string} email - the provided email address
     * @returns boolean | string
     */
or 
/**
     * Applied to slot from SuccessScreen
     */